js/letters.js

```diff
@@ -6,20 +6,28 @@ cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non \
 proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";
 
 var counter = {
-  "a" : 0,  "b" : 0,  "c" : 0,  "d" : 0,
-  "e" : 0,  "f" : 0,  "g" : 0,  "h" : 0,
-  "i" : 0,  "j" : 0,  "k" : 0,  "l" : 0,
-  "m" : 0,  "n" : 0,  "o" : 0,  "p" : 0,
-  "q" : 0,  "r" : 0,  "s" : 0,  "t" : 0,
-  "u" : 0,  "v" : 0,  "w" : 0,  "x" : 0,
-  "y" : 0,  "z" : 0
+  "a": 0, "b": 0, "c": 0, "d": 0,
+  "e": 0, "f": 0, "g": 0, "h": 0,
+  "i": 0, "j": 0, "k": 0, "l": 0,
+  "m": 0, "n": 0, "o": 0, "p": 0,
+  "q": 0, "r": 0, "s": 0, "t": 0,
+  "u": 0, "v": 0, "w": 0, "x": 0,
+  "y": 0, "z": 0
 }
 
-function countLetters(counter, sample_text){
-  // FIX ME
-}
+function countLetters(counter, sample_text) {
+  let text = sample_text.toLowerCase();
+  if (sample_text.length === 0) {
+    return counter;
+  }
+  let letter = text.slice(0, 1);
+  if ([letter]) {
```

Comment:
So this if statement will always return true because this just evaluates to an array. An array regardless of whether it's empty or not is always true. So the approach here is to use Object.hasOwnProperty to figure out if the character you sliced off is in the counter object. This will return a valid true or false based on whether that character exists in the object.